packages/tap-runner/src/tap-helper.ts

```diff
@@ -58,3 +58,27 @@ export async function captureTapResult(tapProcess: ChildProcessWithoutNullStream
   }
   return tapResult;
 }
+
+export function parseArguments(args: string[], hookFile: string, testFile: string): string[] {
```

Comment:
It's not really parsing. How about `buildArguments`?

Reply:
My thoughts was that it parses the template variables, but it indeed also builds the arguments if it is incomplete. I don't know a better name for now than `buildArguments`, but it isn't describing all the functionality in my opinion.